src/Symfony/Component/Serializer/Normalizer/BackedEnumNormalizer.php

```diff
@@ -65,7 +65,7 @@ public function denormalize($data, string $type, string $format = null, array $c
             return $type::from($data);
         } catch (\ValueError $e) {
             if (isset($context['has_constructor'])) {
-                throw new InvalidArgumentException('The data must belong to a backed enumeration of type '.$type);
+                throw new InvalidArgumentException('The data must belong to a backed enumeration of type '.$type, previous: $e);
```

Comment:
5.4 is compatible with PHP 7.2 so we cannot use named arguments

Reply:
```suggestion
                throw new InvalidArgumentException('The data must belong to a backed enumeration of type '.$type, 0, $e);
```